From 6e2f642418cf7089a398be65c1fffe2d7a77cc8c Mon Sep 17 00:00:00 2001
From: Andy Ross <andrew.j.ross@intel.com>
Date: Wed, 11 Jun 2014 20:04:29 -0700
Subject: [PATCH 068/117] atomisp_gmin_platform: Don't create BYT clock device

The vlv2_plat_clock device is already being created in
platform_vlv2_plat_clk.c (the patches crossed).  There's no need to do
it here.

Issue: GMIN-1553
Signed-off-by: Andy Ross <andrew.j.ross@intel.com>
---
 arch/x86/platform/intel-mid/atomisp_gmin_platform.c | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/arch/x86/platform/intel-mid/atomisp_gmin_platform.c b/arch/x86/platform/intel-mid/atomisp_gmin_platform.c
index 89f4905..c914354 100644
--- a/arch/x86/platform/intel-mid/atomisp_gmin_platform.c
+++ b/arch/x86/platform/intel-mid/atomisp_gmin_platform.c
@@ -655,18 +655,3 @@ int camera_sensor_csi(struct v4l2_subdev *sd, u32 port,
         return 0;
 }
 EXPORT_SYMBOL_GPL(camera_sensor_csi);
-
-static int __init gmin_plat_init(void)
-{
-	/* BYT-T output clock driver required by the MIPI-CSI
-	 * camera modules */
-	if (IS_ERR(platform_device_register_simple("vlv2_plat_clk",
-						   -1, NULL, 0)))
-	{
-		pr_err("Failed to register vlv2_plat_clk device");
-		return -ENODEV;
-	}
-	return 0;
-}
-
-device_initcall(gmin_plat_init);
-- 
1.9.3

